index.html

```diff
@@ -0,0 +1,332 @@
+            </section>
+
+            </section>
+
+            <section id="projects">
+
+            </section>
+
+            <about id="about">
+                About
+            </about>
+
+            <p id="aboutP">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+                dolore magna aliqua. Mollis nunc sed id semper risus in. Accumsan tortor posuere ac ut consequat semper
+                viverra.
+            </p>
+
+            <div class="buttons" id="myCV">
+                <a href="#" class="submit-button">Get my CV</a>
+                <img id="getMyCv" src="/img/download.svg" alt="download">
+            </div>
+
+
+
+
+
+            <br><br><br>
+
+
+
+
+
+
+            <div id="colorBoxes">
+                <div class="colorboxes">
+                    <div id="blueBox">
+                        <div class="colorBoxTitle">
+                            Languages
+                        </div>
+                        <ul class="itemsList">
+                            <li> JavaScript (ES5/ES6)</li>
+                            <li> Ruby</li>
+                            <li> HTML</li>
+                            <li> CSS</li>
+                            </li>
+                        </ul>
+                        <img id="hashtagIcon" src="/img/hastag.png" alt="hastagIcon">
+                    </div>
+                    <br>
+
+                    <div id="redBox">
+                        <div class="colorBoxTitle">
+                            Frameworks
+                        </div>
+                        <ul class="itemsList">
+                            <ul class="itemsList">
+                                <li> Bootstrap</li>
+                                <li> Ruby on rails</li>
+                                <li> RSpec</li>
+                                <li> Capibara</li>
+                                <li> Selenium</li>
+                                </li>
+                            </ul>
+                            <img id="squaresRed" class="images" src="/img/squares.svg" alt="squaresIcon">
+                    </div>
+                    <br>
+                    <div id="yellowBox">
+                        <div class="colorBoxTitle">
+                            Skills
+                        </div>
+                        <ul class="itemsList">
+                            <li> Codekit</li>
+                            <li> Codepen</li>
+                            <li> Github</li>
+                            <li> GitLab</li>
+                            <li> Terminal</li>
+                            </li>
+                        </ul>
+                        <img id="lampYellow" class="imagesYellow" src="/img/lamp.svg" alt="lampIcon">
+                    </div>
+
+                </div>
+
+            </div>
+
+
+        </section>
+    </main>
+    <footer>
+    </footer>
+</body>
+
+
```

Comment:
- [ ] kindly consider inclosing the whole about me section in one section and consider aligning content with flexbox this will help you to fulfill the below requirement 👇 
> Make sure the content of the "About myself" section is aligned with flexbox.